.github/workflows/community-label.yml

```diff
@@ -7,7 +7,9 @@ on:
         types: [opened]
 
 concurrency:
-    group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
@adimoldovan Could `github.ref` be replaced with `github.head_ref` to get the PR branch instead of trunk?

Reply:
It would help if we only had the `pull_request_target` event, but we also have the `issues` event for which `github.head_ref` is undefined.
On the other hand the group could be simpler in our case, because we only have the opened action for both events. I went with a group that would cover more actions in case they would be added later.
I guess we could go with the shorter `community-label-${{ github.event_name == 'pull_request_target' && github.head_ref || github.run_id }}`.

What do you think?

Reply:
That makes sense. Looks good as is.